app/models/idea.rb

```diff
@@ -1,3 +1,11 @@
 class Idea < ApplicationRecord
   belongs_to :user
+  has_many :bookings
+  has_many :reviews, through: :bookings
+  validates :title, presence: true
+  validates :description, presence: true
+  validates :category, presence: true
+  validates :start_date, presence: true
+  validates :end_date, presence: true
+  validates :minimum_application_duration, presence: true
```

Comment:
pas indispensable

Reply:
took minimum_application_duration off

Reply:
ligne 6 add' length: { minimum: 100 }' to avoid too short description